scrapy/spidermiddlewares/httperror.py

```diff
@@ -46,6 +46,10 @@ def process_spider_input(self, response, spider):
 
     def process_spider_exception(self, response, exception, spider):
         if isinstance(exception, HttpError):
+            spider.crawler.stats.inc_value('httperror/response_dropped_count')
+            spider.crawler.stats.inc_value(
```

Comment:
Maybe it should use the same "ignored" term as the subsequent INFO message ("Ignoring" is also in the `HttpError` exception message) instead of "dropped".
Besides that small nitpick, looks good to me.